.github/workflows/test.yaml

```diff
@@ -0,0 +1,49 @@
+name: PyTest
+on:
+  push:
+    branches:
+      - "*"
+
+env:
+  TKP_DBNAME: testdb
+  TKP_DBUSER: root
+  TKP_DBPASSWORD: test_pw
+
+jobs:
+  pytest:
+    runs-on: ubuntu-latest
+
+    container:
+      image: quay.io/casacore/casacore
+      options: --user root
+
+    steps:
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install python+pip and PostgreSQL
+        run: |
+          apt-get update
+          apt-get install -y python3 python3-pip
+          apt-get install -y postgresql libpq-dev
+
+      - name: Start sql service
+        run: service postgresql start
+
+      - name: Create db user and install python dependencies
+        run: |
+          su - postgres -c "psql -c 'CREATE DATABASE testdb;'"
+          su - postgres -c "psql -c \"CREATE USER root WITH PASSWORD 'test_pw';\""
```

Comment:
Everything else looks straight forward, but I was wondering if DDL like this could be factored out into separate `.sql` scripts.  Then running the test suite outside of CI could be simpler.  WDYT?